notebooks/thoth_support_github_data.ipynb

```diff
@@ -37,6 +37,7 @@
     "from s3_communication import S3Communication\n",
```

Comment:
<p>Is there any way to check for the other two conditions as well? (i.e. issue is on GH board, and issue has PR linked)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Yes, An (issue) [https://github.com/thoth-station/mi/issues/589] was created to upgrade (MI tool)[https://github.com/thoth-station/mi] so that we can track these features as well. Because of some time constraint, we thought of wrapping up the project by this week so that when the upgrade happens, someone who is working can create a new PR and update the notebook later.

Reply:
<p>Sounds good. Would you mind creating an issue in this repo to add these two criteria to the analysis in this notebook? Just wanna make sure this doesn't get lost in time :)</p>